File: VB/AdditionalCommands.Module.Win/WinModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp

Namespace AdditionalCommands.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class AdditionalCommandsWindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
	End Class
End Namespace

File: CS/AdditionalCommands.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;

namespace AdditionalCommands.Module.Win
{
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class AdditionalCommandsWindowsFormsModule : ModuleBase
    {
        public AdditionalCommandsWindowsFormsModule()
        {
            InitializeComponent();
        }
    }
}
